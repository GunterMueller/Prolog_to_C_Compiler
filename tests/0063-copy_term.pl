main :-
	copy_term(f(X, Y), Z), display(Z), nl,
	copy_term(X, -10), display(X), nl,
	copy_term(f(a, X1), f(X1, b)), display(X1), nl,
	copy_term(f(X2, X2), f(A, B)), display([X2, A, B]), nl,
	\+copy_term(a, 'ok'),
	X3 = f(Z1, Z2), copy_term(X3, U), U \== f(Z1, Z2), display([X3, U]), nl,
	\+((copy_term(f(a, X4), f(X4, b)), copy_term(f(a, X4), f(X4, b)))).
