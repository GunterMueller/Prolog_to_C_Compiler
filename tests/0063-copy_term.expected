f(_4, _5)
-10
a
[_10,_14,_14]
[f(_16, _17),f(_20, _21)]
